contracts/cosmwasm-vm/cw-ibc-core/src/contract.rs

```diff
@@ -31,14 +30,500 @@ impl<'a> CwIbcCoreContext<'a> {
         info: MessageInfo,
         msg: ExecuteMsg,
     ) -> Result<Response, ContractError> {
-        todo!()
+        match msg {
+            ExecuteMsg::RegisterClient {
+                client_type,
+                client_address,
+            } => {
+                self.check_sender_is_owner(deps.as_ref().storage, info.sender.clone())
+                    .unwrap();
+                let client_type = ClientType::new(client_type);
+                self.register_client(deps, client_type, client_address)
+            }
+            ExecuteMsg::CreateClient {
+                client_state,
+                consensus_state,
+                signer,
+            } => {
+                self.check_sender_is_owner(deps.as_ref().storage, info.sender.clone())
+                    .unwrap();
+
```

Comment:
lets map it to contract error. Due to poor debuggability of contracts its better to have more verbose error handling.